9c-internal/9c-pluggable/application.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: 9c-pluggable
+  namespace: argocd
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/planetarium/9c-infra.git
+    targetRevision: 9c-pluggable
```

Comment:
here it means which branch you want to sync with argocd.
will you use `9c-pluggable` branch after this PR? otherwise it should be main branch